subprojects/matrix-benchmarking-workloads/rhods-notebooks-ux/store.py

```diff
@@ -846,6 +850,24 @@ def _parse_directory(fn_add_to_matrix, dirname, import_settings):
 
     print("parsing done :)")
 
+def _parse_start_end_times(dirname):
+    with open(dirname / '_ansible.log', 'rb') as file:
+        first = file.readline().decode('utf-8').strip()
+        file.seek(-20, 2)
+        while file.read(1) != b'\n':
+            file.seek(-2, 1)
```

Comment:
🤔 
I guess it would be more readable to open the file in `utf-8` (remove `rb`), read all the lines, and parse the first and last. This seek might be efficient for a big file, but here the log file won't be more than a few hundreds KB